source/lex.tex

```diff
@@ -1068,18 +1081,19 @@
 \indextext{suffix!\idxcode{U}}%
 \indextext{suffix!\idxcode{l}}%
 \indextext{suffix!\idxcode{u}}%
-The type of an integer literal is the first of the corresponding list
-in \tref{lex.icon.type} in which its value can be
-represented.
+Using the optional \grammarterm{integer-suffix},
+the type of an \grammarterm{integer-literal} is
+the first of the corresponding list in \tref{lex.icon.type}
+in which its value can be represented.
+An \grammarterm{integer-literal} is a prvalue.
```

Comment:
I find this phrasing a little awkward. How about:

> The type of an *integer-literal* is the first type in the list in \tref{lex.icon.type} corresponding to its optional *integer-suffix* in which its value can be represented.

Reply:
Yes, that's better.

Reply:
Fixed.